### DISCUSSION TITLE
Hoover Washerdryer

### DISCUSSION THREAD

**Discussion by RubenKelevra** [ofalvai/home-assistant-candy#49] [General]
Hey guys,

the Haier group also offers devices under the Hoover brand, so I was wondering if they are similar enough to be supported?

I got the HDPD696AMBC/1-S Model.

I tried the CandySimplyFi-tool but it shows only a connection error since port 80 is closed on the device.

**Comment by ofalvai** [comment]
I think Hoover devices have the same API and should work with this integration. I have a Candy washing machine and when I downloaded the Hoover app on my phone, it was working with my washing machine.

**Comment by RubenKelevra** [comment]
I use the [hOn app](https://play.google.com/store/apps/details?id=com.haiereurope.hon). Is this the app you're referring to?

Any idea why I can't connect locally to the machine? :/

**Comment by ofalvai** [comment]
Sorry, I was referring to the Hoover Wizard app: https://play.google.com/store/apps/details?id=it.hoover.wizard

Maybe this hOn app is using a completely different protocol and that's why CandySimplyFi-tool doesn't work with your model.

**Comment by ofalvai** [comment]
I'm closing this issue to clean up open issues, but feel free to reopen it with new information if you think this integration can/should work with your model.

**Comment by RubenKelevra** [comment]
Look what I found:

![IMG_20211125_095508.jpg](https://user-images.githubusercontent.com/614929/143479380-4cef715c-7c06-497f-961c-6f40e9038199.jpg)

![IMG_20211125_095448.jpg](https://user-images.githubusercontent.com/614929/143479393-a2015a57-59e1-4624-83f2-63f6691b5536.jpg)

![IMG_20211125_095452.jpg](https://user-images.githubusercontent.com/614929/143479402-cd29b0c9-9956-41b7-a746-d7ea9a8dccce.jpg)

And they actually use an ESP32-Solo-1 for the WiFi-Stuff (:

![IMG_20211125_095431.jpg](https://user-images.githubusercontent.com/614929/143479454-b03d781f-c42f-4c60-9dd9-03da7aca0ee5.jpg)

Looks pretty interesting for me to tap into, but don't have any experience with that. But I got a handful USB-to-TTL adapters and knowledge of Linux.

What you think? 🤔

**Comment by RubenKelevra** [comment]
> I'm closing this issue to clean up open issues, but feel free to reopen it with new information if you think this integration can/should work with your model.

I don't have enough rights to reopen issues.

**Comment by ofalvai** [comment]
Apologies, I have converted the issue now into a discussion.

Regarding the ESP board: it's a pretty cool discovery, it would be great to see this reverse enginnered. Although this is way beyond the scope of this integration that uses the local API :)

**Reply by maury77** [reply]
Any idea for Integration hon Device?
